src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -722,7 +723,7 @@ def unsubscribe_all_events(self) -> None:
         self.subscriptions.unsubscribe_all()
 
     def add_on_child_created(
```

Comment:
This is the `add_on_child_created` method of the service class which has a different implmentation between PyFluent, old PyConsole and new PyConsole. In this function the callback takes the child path.

For the new PyConsole implementation, the callback function doesn't require any transformation, it can be passed directly to PyStateEngine/C++.

We need to update the old PyConsole implmentation to transform the callback function to take a PyStateEngine's ModelState object.